localization/pt/caching/README.md

```diff
@@ -0,0 +1,326 @@
+---
+title: Caching
```

Comment:
This should be translated to Portugese as well

Reply:
Changed to Cache.